WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/WebView.swift

```diff
@@ -54,12 +58,18 @@ struct WebView: UIViewRepresentable {
     /// A url to trigger dismissing of the web view.
     let urlToTriggerExit: String?
 
+    /// Optional closure for when a web page loads successfully.
+    let pageLoadHandler: ((URL) -> Void)?
```

Comment:
Super nit: I belive we call this closure only for the `WebView`'s given `url` and not for any further URLs from user initiated navigations. Should we update the comment to explain that?

Reply:
I don't think so, this closure is triggered by the delegate method `func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!)` which is called when a new page is loaded (redirect success). Can you explain further if I'm misunderstand something here? @selanthiraiyan

Reply:
Good point - updated the comments in 0d62760b81b951e1df147ceee1a483d1da5da8b2